pkg/config/runtime_setting.go

```diff
@@ -0,0 +1,72 @@
+/// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Could we had tests for this and runtime_setting_log_level.go ?

Reply:
done